### PR TITLE
Add up/down badges for Heroku to README.md

### LINKED ISSUE #672: Add heroku install version badges to readme
We have two heroku installs: `mutual-aid-demo` and `mutual-aid-test`. Would be great for our [README.md] to show badges indicating up/down status and deployed versions.

### PR DESCRIPTION
**Pull request by Shaglock**
## Why
Resolves #672. 
### Pre-Merge Checklist
- [x] All new features have been described in the pull request
- [x] Security & accessibility have been considered
- [x] All outstanding questions and concerns have been resolved
- [ ] Any next steps that seem like good ideas have been created as issues for future discussion & implementation
- [x] High quality tests have been added, or an explanation has been given why the features cannot be tested
- [x] New features have been documented, and the code is understandable and well commented
- [ ] Entry added to CHANGELOG.md if appropriate

## What
So I basically added 2 badges for up(deployed) status of the apps.

image

## How

I was using [this repo](https://github.com/pussinboots/heroku-badge) for the badges, I was also considering [this](https://github.com/chrisjh/heroku-shields), but the first one looks better to me, as it is more recent, has more options, and doesn't show 'up' on apps that do not exist 😆. 

## Next Steps
Probably we need to find a solution for version deployed badge, maybe move to separate issue or close the issue if it's not possible.

## Outstanding Questions, Concerns and Other Notes
I am not sure that we can show deployed version, at least I didn't find any solution for that (if we can please tell me how).

### CODE REVIEW

**Comment by exbinary** [comment]
@Shaglock , thanks for looking into this! 🙌🏾

The up/down status is nice, but TBH the deployed version is actually the more useful bit.

I wonder if it would be worth adding a simple endpoint in the app itself that could read the deployed `version` from `package.json`. This could be paired with a service like [Badgen](https://badgen.net/) (there are others too) which will do the work of generating the badge itself. Badgen has [/https](https://badgen.net/https) endpoint support that might work?

Of course these are nice-to-haves, so only if you're curious/interested :)

**Comment by Shaglock** [comment]
Hey @exbinary! Thank you for your review. I created an endpoint as you suggested and added badges for version. But I think we need to somehow distinguish badges because at the moment they have the same text for demo and test Heroku apps, maybe we need to add some text to the badge, or different links on badges is enough?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 [![CircleCI](https://circleci.com/gh/rubyforgood/mutual-aid/tree/main.svg?style=svg)](https://circleci.com/gh/rubyforgood/mutual-aid/tree/main)
+[![Heroku Mutual Aid Test Status](http://heroku-badge.herokuapp.com/?app=mutual-aid-test&svg=1)](https://mutual-aid-test.herokuapp.com)
+[![Heroku Mutual Aid Demo Status](http://heroku-badge.herokuapp.com/?app=mutual-aid-demo&svg=1)](https://mutual-aid-demo.herokuapp.com)
+[![Heroku Mutual Aid Test Version](https://badgen.net/https/mutual-aid-test.herokuapp.com/version)](https://mutual-aid-test.herokuapp.com)
+[![Heroku Mutual Aid Demo Version](https://badgen.net/https/mutual-aid-demo.herokuapp.com/version)](https://mutual-aid-demo.herokuapp.com)
```
Comment (exbinary) [inline README.md:5]: One thought is we could try a markdown table to give the badges meaningful descriptors?
Something like this:
```suggestion
|CircleCI|Demo site|Test site|
|-|-|-|
|[![CircleCI](https://circleci.com/gh/rubyforgood/mutual-aid/tree/main.svg?style=svg)](https://circleci.com/gh/rubyforgood/mutual-aid/tree/main)|[![Heroku Mutual Aid Demo Status](http://heroku-badge.herokuapp.com/?app=mutual-aid-demo&svg=1)](https://mutual-aid-demo.herokuapp.com) [![Heroku Mutual Aid Demo Version](https://badgen.net/https/mutual-aid-demo.herokuapp.com/version)](https://mutual-aid-demo.herokuapp.com)|[![Heroku Mutual Aid Test Status](http://heroku-badge.herokuapp.com/?app=mutual-aid-test&svg=1)](https://mutual-aid-test.herokuapp.com) [![Heroku Mutual Aid Test Version](https://badgen.net/https/mutual-aid-test.herokuapp.com/version)](https://mutual-aid-test.herokuapp.com)|
```

**Review by exbinary — APPROVED** [review]
This is sweet @Shaglock ! 🍮

Added one idea for the labeling problem but either way, thanks for implementing this!

File: README.md (hunk 1 above)
Comment (Shaglock) [reply README.md:5]: Great idea!

**Comment by exbinary** [comment]
@Shaglock , looks like https://heroku-badge.herokuapp.com/ is down and i'm not confident it'll be fixed soon or maintained going forward. I'm going to remove that link the readme until it goes back up. The version was the more important part anyway!

**Pull request merged**